modules/osdk-cco-aws-sts-enabling.adoc

```diff
@@ -95,8 +95,9 @@ var in = minterv1.AWSProviderSpec{
            },
            Effect:   "Allow",
            Resource: "arn:aws:s3:*:*:*",
-       },
-   },
+   	 },
+	},
+	STSIAMRoleARN: "<role_arn>",
```

Comment:
@fxierh Should this be a user-replaceable value like this (`<role_arn>`) or verbatim `ROLE-ARN` as shown in https://github.com/openshift/openshift-docs/pull/64808#discussion_r1349475000 ?

Reply:
The former.

Reply:
The former.